app/controllers/registered_apps_controller.rb

```diff
@@ -0,0 +1,47 @@
+class RegisteredAppsController < ApplicationController
+
+  def show
+    @registered_app = RegisteredApp.find(params[:id])
+  end
+
+  def new
+    @user = User.find(params[:user_id])
+    @registered_app = RegisteredApp.new
+  end
+
+  def create
+
+    @user = User.find(params[:user_id])
+    @registered_app = RegisteredApp.new
+    @registered_app.name = params[:registered_app][:name]
+    @registered_app.url = params[:registered_app][:url]
+    @registered_app.user_id = @user.id
```

Comment:
could you change this to something like `@registered_app = RegisteredApp.new(app_params)`?